### PR TITLE
Add: Patient create form with redux

### PR DESCRIPTION
**Pull request by crazieeKinG**

### CODE REVIEW

**Review by aman007shrestha — APPROVED** [review]
Well written. Looks good to me.

**Review by beingbiplov — APPROVED** [review]

File: src/components/Patient/CreatePatientForm.tsx [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+        <Form
+            labelAlign="left"
+            labelCol={{ span: 8 }}
+            onFinish={handleForm}
+            className="margin-top"
+        >
+            <Item
+                label="First name"
+                name="firstName"
+                rules={[{ required: true, message: "Please fill this field!" }]}
```
Comment (RikLakhe) [inline src/components/Patient/CreatePatientForm.tsx:80]: if the message is similar through out the app, we can have it maintained in config file 
/src/config/appConfig.js 

FORM_DEFAULT_REQUIRED_MESSAGE = "Please fill this field!"

File: src/components/Patient/CreatePatientForm.tsx [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+            >
+                <Input placeholder="Enter your ethnicity" />
+            </Item>
+            <Item
+                label="Gender"
+                name="gender"
+                rules={[
+                    {
+                        required: true,
+                        message: "Please fill this field!",
```
Comment (RikLakhe) [inline src/components/Patient/CreatePatientForm.tsx:111]: This should be the selected? is it not ?

File: src/components/Patient/CreatePatientForm.tsx [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+                name="document"
+                label="Document Image"
+            >
+                <Upload {...uploadProps}>
+                    <Button>Upload</Button>
+                </Upload>
+            </Item>
+
+            <Item name="aggrement" valuePropName="checked">
+                <Checkbox onChange={handleAggrement}>I Agree</Checkbox>
```
Comment (RikLakhe) [inline src/components/Patient/CreatePatientForm.tsx:212]: Let's have a logic where until we check this we cannot submit and upon clicking the submit button we are shown some error message below this

**Pull request merged**